its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/JreProvisioningTest.java

```diff
@@ -134,4 +134,38 @@ void jreProvisioning_endToEnd_cacheHit_reusesJre() throws IOException {
       "JreResolver: Cache miss",
       "Starting the Java Runtime Environment download.");
   }
+
+  @Test
+  void jreProvisioning_endToEnd_parameters_propagated() throws IOException {
+    var projectDir = TestUtils.projectDir(basePath, PROJECT_NAME);
+
+    SonarCloudUtils.runBeginStep(
+      projectDir,
+      SONARCLOUD_PROJECT_KEY,
+      new StreamConsumer.Pipe(new StringWriter()),
+      "/d:sonar.scanner.os=windows",
+      "/d:sonar.scanner.arch=x64",
+      "/d:sonar.scanner.skipJreProvisioning=true",
+      "/d:sonar.scanner.connectTimeout=42",
+      "/d:sonar.scanner.socketTimeout=100",
+      "/d:sonar.scanner.responseTimeout=500",
+      "/d:sonar.userHome="+projectDir.toAbsolutePath());
```

Comment:
Also sonar.scanner.sonarcloudUrl, sonar.scanner.apiBaseUrl and sonar.scanner.javaExePath

Reply:
I will add the URLs, but the javaExePath would be different per user and in the pipeline, so I will skip it.